components/Headers/Header.module.css

```diff
@@ -2,6 +2,7 @@
   color: var(--accent-color-orange);
```

Comment:
Better name it primary or secondary instead of orange, maybe you want to change it sometime to green or red then you'd need to change the name again.

Reply:
Right :/